numpy/polynomial/polyutils.py

```diff
@@ -690,6 +690,39 @@ def _fit(vander_f, x, y, deg, rcond=None, full=False, w=None):
         return c
 
 
+def _pow(mul_f, c, pow, maxpower):
+    """
+    Helper function used to implement the ``<type>pow`` functions.
+
+    Parameters
+    ----------
+    vander_f : function(array_like, int) -> ndarray
+        The 1d vander function, such as ``polyvander``
+    pow, maxpower :
+        See the ``<type>pow`` functions for more detail
+    mul_f : function(array_like, array_like) -> ndarray
+        The ``<type>mul`` function, such as ``polymul``
+    """
+    # c is a trimmed copy
+    [c] = as_series([c])
+    power = int(pow)
```

Comment:
Do you want to change this to require integers as you have done elsewhere? I suppose power is different from degree in that it can be float in other circumstances.

Reply:
Not in this PR, but I thought about that. I'll let you decide.

Reply:
Let's leave it for later.